src/GameLogic.cpp

```diff
@@ -59,6 +60,20 @@ void GameLogic::pollEvent(sf::RenderWindow *App, sf::Clock gameTime, double targ
 									}
 								}
 								break;
+							case GameState::State::SUCCESS:
+								mousePosition = sf::Mouse::getPosition(*App);
+								if (mousePosition.x >= 0 && mousePosition.x <= 800 && mousePosition.y >= 0 && mousePosition.y <= 600){
```

Comment:
These checks aren't needed because when the mouse clicks off of our screen, the screen will unfocus & not register as an event, so you can remove this